### PR TITLE
Reusuable ClearDeckAlertModal

### PR DESCRIPTION
**Pull request by Kadee80**
## overview

This PR addresses #1236 (checkboxes 2-4) UI only. It does not involve checking for a token or pipette.

Enables [calibrate] button in the `CalibrationCard`, links to route: `/robots/${name}/deck-calibration/step-1` which renders the now reusable ClearDeckAlertModal only at this time. 

Cancel button returns to robot settings continue is null.

## changelog

- Show clear deck alert as default deck calibration screen
- Update ClearDeckAlert Props
    - Note: ActualPipette is something very change pipette specific, in the future factor this out as props.note or something more reusable
- Rename `CalibateDeck/` directory to `DeckMap/` - this was confusing because this folder only generated a DeckMap for calibration and we need `CalibrateDeck` for deck calibration.
- Replace duplicate `alert` icon (same as `alert-circle`), rename `warning` icon to `alert`

## review requests
This should not affect the PD at all.  Please check that ClearDeckAlertModal behaves as when clicking [change/attach] pipettes or [calibrate] buttons on the RobotSettings page.

### CODE REVIEW

File: app/src/components/ClearDeckAlertModal/styles.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.alert_list {
```
[the remark this answers is not in the document]
Comment (IanLondon) [reply app/src/components/ClearDeckAlertModal/styles.css]: Although the modal doesn't look weird or anything :grey_question:

File: app/src/components/ClearDeckAlertModal/index.js [hunk 2]
```diff
@@ -2,26 +2,31 @@
 import * as React from 'react'
 import {Link} from 'react-router-dom'
 
-import type {ChangePipetteProps} from './types'
+import type {PipetteConfig} from '@opentrons/labware-definitions'
 import {AlertModal} from '@opentrons/components'
 import styles from './styles.css'
 
+type Props = {
+  actualPipette?: ?PipetteConfig,
```
[the remark this answers is not in the document]
Comment (Kadee80) [reply app/src/components/ClearDeckAlertModal/index.js]: 👍

File: app/src/components/DeckMap/LabwareItem.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ export default function LabwareItem (props: LabwareItemProps) {
   const item = (
     <LabwareContainer width={width} height={height} highlighted={highlighted}>
       <g className={plateClass}>
-        <Plate containerType={type} />
+        <Plate containerType={type} wellContents={{}} />
```
[the remark this answers is not in the document]
Comment (Kadee80) [reply app/src/components/DeckMap/LabwareItem.js]: yeah i had a gnarly merge conflict when I stash popped with the renameing. good catch.

File: app/src/components/ClearDeckAlertModal/styles.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcous) [reply app/src/components/ClearDeckAlertModal/styles.css]: @IanLondon is right, the import is definitely required (I was trying this out the other day to confirm). That being said, with these imports we end up with _a lot_ of duplication in our final CSS, due to what I assume is interplay between postcss-import and CSS modules.

Not sure what the appropriate solution is, but maybe it'll be as easy as running it through a CSS minifier?

**Pull request merged**